src/Evolution/Systems/GeneralizedHarmonic/Tags.hpp

```diff
@@ -0,0 +1,111 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+namespace GeneralizedHarmonic {
+template <size_t Dim>
+struct SpacetimeMetric : db::DataBoxTag {
+  using type = tnsr::aa<DataVector, Dim, Frame::Grid>;
+  static constexpr db::DataBoxString_t label = "SpacetimeMetric";
+};
+template <size_t Dim>
+struct Pi : db::DataBoxTag {
+  using type = tnsr::abb<DataVector, Dim, Frame::Grid>;
```

Comment:
`Pi` is just `tnsr::aa`, no? The conjugate momentum is of the same tensor type as the evolved variable since it is just a temporal derivative.

Reply:
Correct (although it is not a straight time derivative)

Reply:
TRUE FACT :smile_cat: